sample/sample8-middlewares/UserController.ts

```diff
@@ -0,0 +1,56 @@
+import {Request} from "express";
+import {Controller} from "../../src/decorator/controllers";
+import {Get, Post, Put, Patch, Delete, Use} from "../../src/decorator/methods";
+import {Req, Next} from "../../src/decorator/params";
+
+@Controller()
+export class UserController {
+
+
+    @Use()
+    middleware( @Next() next: any) {
```

Comment:
if was afraid that this middleware will be called for `all route in all controllers` because it will lead to lot of problems. When you have user module with its own controller and lets say posts controller in the posts module and it brings *global* middleware which changes something, and you need to think why user controller actions works unexpectedly. If this middleware is for all controllers, then it should not be in specific controller.